doctrine.rst

```diff
@@ -23,8 +23,8 @@ code:
 
 .. code-block:: terminal
 
-    composer require doctrine
-    composer require maker --dev
+    $ composer require doctrine/doctrine-bundle
```

Comment:
By the way, why don't we use `orm` here instead? With this step the required `doctrine/orm` package would be missing leading to errors.